woocommerce/woocommerce
# Update states.php -> added Senegal Regions

### All Submissions:






### Changes proposed in this Pull Request:



Cheery picked the commit from https://github.com/woocommerce/woocommerce/pull/29844 and created a new PR as the repo structure changed too much, a rebase probably won't work. 






### How to test the changes in this Pull Request:



1. Switch to this PR branch.
2. Verify state list is available on Checkout page after selecting Senegal as shipping destination Country.





### Other information:




### FOR PR REVIEWER ONLY:
